### ISSUE TITLE
`EGL_EXT_image_dma_buf_import`: advertise which hints are supported

### ISSUE THREAD

**Issue by emersion** [KhronosGroup/EGL-Registry#173]
`EGL_EXT_image_dma_buf_import` has a number of hints:

```
        EGL_YUV_COLOR_SPACE_HINT_EXT    0x327B
        EGL_SAMPLE_RANGE_HINT_EXT       0x327C
        EGL_YUV_CHROMA_HORIZONTAL_SITING_HINT_EXT  0x327D
        EGL_YUV_CHROMA_VERTICAL_SITING_HINT_EXT    0x327E
```

However there is no way for an EGL client to figure out which hints will be taken into account by the driver, and which will be ignored. It would be nice to allow EGL clients to query this information.

Ref https://gitlab.freedesktop.org/wayland/wayland-protocols/-/merge_requests/153

**Comment by stonesthrow** [comment]
Sounds interesting. Whatever you work out with freedesktop will have that support. From that I guess you can decide if a new EGL  extension is needed or a version 2 of EGL_EXT_image_dma_buf_import_modifiers?

**Comment by emersion** [comment]
Right. The naive solution would be to just add a new empty `EGL_MESA_image_dma_buf_import_hints` extensions with the only purpose of indicating that the hints are supported and taken into account by the driver.

**Comment by stonesthrow** [comment]
Sounds like a start to a proposal. Bounce a proposal off MESA/FreeDesktop people since its your expertise. I and Jon will review for a good extension. thx
